test/test_helper.rb

```diff
@@ -15,6 +15,21 @@
   add_filter '/config/'
 end
 
+module Coverband::Test
+  def self.reset
```

Comment:
@danmayer pulled this into a separate class since rails full stack test does not seem to hit MinitTest::Test#setup.